### ISSUE TITLE
How to fetch accelerometer data after the hack?

### ISSUE THREAD

**Issue by uditunpluged** [dimitrovskif/hackband#1]
Please help me, how can i extract accelerometer data from sony's smartband swr10.

**Comment by dimitrovskif** [comment]
It's still in progress. There is not much demand for this :)

**Comment by uditunpluged** [comment]
I tried to generate a hacked app using your guidelines, didn't succeed in that. SWR 10 has no api which makes this device limited and dumb. Wish if there was some sdk for it, marvels can be made. :)

**Comment by tyeth** [comment]
Not dumb, merely a black box ;)
Fantastic work as the sentiment of the previous poster implied,
please help us to use this with a small hint in the correct direction others can pickup the torch :)

**Comment by dimitrovskif** [comment]
@uditunpluged @tyeth Use this https://github.com/fbarriga/sony-smartband-open-api

**Comment by dimitrovskif** [comment]
I am busy at the moment but I am working on a fork of `sony-smartband-open-api`. The open API has invalid accelerometer implementation (the int32 parser is wrong), but I managed to fix it and I've achieved 20fps floating point data that contains high precision +-1g measurements.

There are also more accelerometer modes depending on app (ex. games have fast updates, fitness trackers have slow but accurate updates)
